index.html

```diff
@@ -0,0 +1,75 @@
+      </p>
+
+      <form id="survey-form">
+        <div class="form-control">
+          <label for="name" id="name-label">Name</label>
+          <input type="text" name="name" placeholder="Enter your name" required />
+        </div>
+
+        <div class="form-control">
+          <label for="email" id="email-label">Email</label>
+          <input type="email" name="email" placeholder="Enter your email" required />
+        </div>
+
+        <div class="form-control">
+          <label for="age" id="label-age">Age(Optional)</label>
+          <input type="number" name="number" placeholder="Enter your age" required />
+        </div>
+
+        <div class="form-control">
+          <label for="role">Which option best describes your current role?</label>
+          <select id="role">
+            <option value="student">Student</option>
+            <option value="full-time-learner">Full time learner</option>
+            <option value="full-time-job">Full time job</option>
+            <option value="prefer-not-to-say">Prefer not to say</option>
+            <option value="other">Other</option>
+          </select>
+        </div>
+
+        <div class="form-control">
+          <label>Would you recommend my channel to a friend?</label>
+          <label for="definitely"><input type="radio" name="recommend-1" value="definitely" id="definitely" checked /> Definitely</label>
+          <label for="maybe"><input type="radio" name="recommend-2" value="maybe" id="maybe" /> Maybe</label>
+          <label for="not-sure">  <input type="radio" name="recommend-3" value="not-sure" id="not-sure" /> Not sure</label>
+        </div>
+
+        <div class="form-control">
+          <label for="feature">What's your favourite feature?</label>
+          <select name="feature" id="feature">
+            <option value="challenges">Challenges</option>
+            <option value="projects">Projects</option>
+            <option value="community">Community</option>
+            <option value="open-source">Open Source</option>
+          </select>
+        </div>
+
+        <div class="form-control">
+          <label>What would you like to see improved?<small>Check all that apply</small></label>
+          <label for="video-length"><input type="checkbox" id="video-length" name="video-length" value="video-length" /> Video length</label>
+          <label for="content-quality"><input type="checkbox" id="content-quality" name="content-quality" value="content-quality" /> Content quality</label>
+          <label for="more-videos"><input type="checkbox" id="more-videos" name="more-videos" value="more-videos" /> Add more videos</label>
+        </div>
+
+        <div class="form-control">
+          <label for="comments">Any comments or suggestions?</label>
+          <textarea rows="4" cols="50" name="comments" id="comments" >Enter your comment here...</textarea>
+        </div>
+
+        <button type="submit" value="submit">Submit</button>
+
+      </form>
```

Comment:
Please the value of the "for" attribute of the label element must be ID of a non-hidden form control.